packages/frontend/web-ui/src/auth/pages/Register.spec.jsx

```diff
@@ -285,4 +285,42 @@ describe(Register.name, () => {
       expect(userCreateOK).toBe(messageConfirmation);
     });
   });
+
+  describe('when called, and click on the button Create', () => {
```

Comment:
I would prefer some pasive sentece as `when called, and the create button is pressed`

Reply:
Done!